src/cinder/audio/OutputNode.cpp

```diff
@@ -96,7 +98,21 @@ OutputDeviceNode::OutputDeviceNode( const DeviceRef &device, const Format &forma
 	mDidChangeConn = mDevice->getSignalParamsDidChange().connect( bind( &OutputDeviceNode::deviceParamsDidChange, this ) );
 
 	mInterruptionBeganConn = Context::deviceManager()->getSignalInterruptionBegan().connect( [this] { disable(); } );
-	mInterruptionEndedConn = Context::deviceManager()->getSignalInterruptionEnded().connect( [this]( bool shouldResume ) { enable(); } );
+	mInterruptionEndedConn = Context::deviceManager()->getSignalInterruptionEnded().connect( [this]( bool resumeImmediately ) {
```

Comment:
Hmm.. So if we can rely that the application is instantiated before DeviceManager, and doesn't ever change, then we could just connect to the `signalDidBecomeActive` signal once.

Removing `resumeImmediately` from the API would certainly make app code simpler.. Currently I need another `signalDidBecomeActive` listener in my app, just to fade the audio up.. Would be simpler to just wait to emit `signalInterruptionEnded`.